documentation/source/library_reference.rst

```diff
@@ -64,6 +64,35 @@ Interacting with the Simulator
 
 .. autofunction:: cocotb.decorators.RunningTask.kill
 
+HDL Datatypes
+-------------
+
+These are a set of datatypes that model the behavior of common HDL datatypes.
+They can be used independently of cocotb for modeling and will replace :class:`BinaryValue`
+as the types used by cocotb's `simulator handles <#simulation-object-handles>`_.
+
+.. versionadded:: 2.0
+.. inheritance-diagram:: cocotb.types
```

Comment:
```suggestion

.. inheritance-diagram:: cocotb.types
```
It's not showing up, assuming it's because of the missing newline.

Reply:
Just seems like it isn't working. I don't see any warnings either.

Reply:
I think because we aren't defining the types in `__init__`. Sphinx ignores imported things. Maybe if I add them to an `__all__` variable it will show up.